www/handleBrowserBackButton.js

```diff
@@ -49,6 +59,13 @@ const handleBrowserBackButton = () => {
   window.addEventListener("popstate", () => {
     const targetTab = window.location.hash.replace("#", "");
     gotoTab(targetTab);
+    const targetNavbarATag = document.querySelector(
+      `a[data-value="${targetTab}"]`
+    );
+
+    // Color the active navbar list item
+    const targetNavbarLi = targetNavbarATag?.parentElement;
```

Comment:
I had to move this outside the `gotoTab` function, otherwise the `map` view will not load (will stuck in the loading screen).